incubator/command-log/pom.xml

```diff
@@ -85,22 +90,6 @@
         <groupId>org.jasig.maven</groupId>
         <artifactId>maven-notice-plugin</artifactId>
       </plugin>
-      <plugin>
-        <groupId>${project.groupId}</groupId>
-        <artifactId>flyweight-maven-plugin</artifactId>
-        <version>${project.version}</version>
-        <configuration>
-          <scopeNames>core amqp::stream mqtt::stream kafka::stream http::stream sse::stream proxy::stream</scopeNames>
-          <packageName>io.aklivity.zilla.runtime.command.log.internal.types</packageName>
-        </configuration>
-        <executions>
-          <execution>
-            <goals>
-              <goal>generate</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
Please keep the generated flyweights local to the module using them.

Java Virtual Machine escape analysis can then better determine only one usage of them inside the same Java module and can then apply better optimizations for these high performance codepaths as desired.